CppXAMLSample/XamlIslandsSample/Application.manifest

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<assembly manifestVersion="1.0" xmlns="urn:schemas-microsoft-com:asm.v1" xmlns:asmv3="urn:schemas-microsoft-com:asm.v3">
+  <compatibility xmlns="urn:schemas-microsoft-com:compatibility.v1"> 
+    <application> 
+      <!--This Id value indicates the application supports Windows 10 functionality -->
+      <supportedOS Id="{8e0f7a12-bfb3-4fe8-b9a5-48fd50a15a9a}" />
+      <!-- 
+        WARNING - DO NOT CHANGE THE maxversiontested VALUE 
+        Microsoft internal - see https://task.ms/33672605

```

Comment:
bug is closed by design. I think you should add the other values here and remove the comment